app/controllers/employers_controller.rb

```diff
@@ -0,0 +1,9 @@
+class EmployersController < InheritedResources::Base
+
+  private
+
+    def employer_params
+      params.require(:employer).permit(:email, :name, :position, :privatenumber)
+    end
+
+end
```

Comment:
Check the indentation in this file.

Also, you can use a linter, like rubocop, which provides a set of rules with the common rules or styles used in the ruby community